src/patternfly/components/Select/select-typeahead.hbs

```diff
@@ -1,4 +1,4 @@
-<ul class="pf-c-select__menu{{#if select--modifier}} {{select--modifier}}{{/if}}"
+<ul class="pf-c-select__menu{{#if select-menu-modifier}} {{select-menu-modifier}}{{/if}}"
```

Comment:
```suggestion
<ul class="pf-c-select__menu{{#if select-menu--modifier}} {{select-menu--modifier}}{{/if}}"
```

Reply:
the format for these names are `hbs-file-name--modifier` (for css classes) and `hbs-file-name--attribute` (for html attributes)